File: EJERCICIOS DE LOS ALUMNOS/NOEMI/08-09-01-26/fibonacci.py
#fibonacci=[0,1]

#for i in range(10):
   # fibonacci.append(fibonacci[-1] + fibonacci[-2])
    
#print(fibonacci)

a=0
b=1
c=0
print(a)
for x in range(11):
    a=b
    b=c
    c=a+b
    print(c)

File: EJERCICIOS DE LOS ALUMNOS/JAVI/JAVA/ProgSistemas/src/Bucles/fibonacci.java
package Bucles;

import java.util.Scanner;

public class fibonacci {

	public static void main(String[] args) {
		
		Scanner teclado = new Scanner(System.in);
		System.out.print("Introduzca el límite de la sucesión: ");
		int num = teclado.nextInt();
		
		int a = 0;
		int b = 1;
		
		for(int i=1; i <= num; i++) {
			
			System.out.println(a); 
			
			int c = a + b;
			a = b;
			b = c;
			
		}
			
		/*
		System.out.println("*************");
		
		int contador = 0;
		
		while(contador < num - 2) {
						
			int c = a + b;			
			a = b;
			b = c;
			
			System.out.println(c); 
			
			contador++;
			
		}
		*/
		
		teclado.close();
	}
	
}
